pkg/apis/pingcap/v1alpha1/tidbinitializer_types.go

```diff
@@ -71,13 +71,17 @@ type TidbInitializerSpec struct {
 
 	// InitSqlConfigMapName reference a configmap that provide init-sql, take high precedence than initSql if set
 	// +optional
-	InitSqlConfigMap *corev1.LocalObjectReference `json:"initSqlConfigMap,omitempty"`
```

Comment:
It would be more extensible to use a struct, for example, we could support reference configmap cross namespaces by adding `namespace` to the object, that is, change `LocalObjectReference` to `ObjectReference`

Reply:
A Pod cannot mount a configmap from other ns, from the definition of `ConfigMapVolumeSource`, only `LocalObjectReference` is included.